engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -728,7 +728,9 @@ class Compiler(
           diagnostics.map(_._2.collect { case e: IR.Error => e }.length).sum
         val warnCount =
           diagnostics.map(_._2.collect { case e: IR.Warning => e }.length).sum
-        println(s"Aborting due to ${count} errors and ${warnCount} warnings.")
+        context.getOut.println(
```

Comment:
@kustosz I actually wonder if we should print this message at all? Seems spurious?